components/post_view/post_message_view/post_message_view.jsx

```diff
@@ -168,6 +168,8 @@ export default class PostMessageView extends React.PureComponent {
                 text={message}
             >
                 <div
+                    aria-readonly='true'
+                    tabIndex='0'
```

Comment:
Might be a dumb question, but wouldn't this apply globally throughout the page? How does it work with menus and when we want to tab through them?

Reply:
`tabIndex` just makes the element focusable, `0` does not place the element in a specific flow order, the flow order is inherited by the element's position on the page.